src/com/selftest/trafficlight/TrafficLightColour.java

```diff
@@ -0,0 +1,17 @@
+package com.selftest.trafficlight;
+
+public enum TrafficLightColour {
+	RED(1), GREEN(1), AMBER(1);

```

Comment:
Wow, you don't get much time on that green light, do you?

Good move to the more accurate AMBER instead of YELLOW :+1: